### ISSUE TITLE
ASSJSON could include metadata

### ISSUE THREAD

**Issue by bensewell** [assimp/assimp#4345]
**Is your feature request related to a problem? Please describe.**
When reviewing ASSIMP conversions, the native JSON format (ASSJSON) is very convenient to use. However, it does not include metadata attached to nodes. This means that data available to ASSIMP is not exposed to the reader of the JSON.

**Describe the solution you'd like**
I'd like the JSON format to include the metadata.

**Additional context**
I am happy to prepare a pull request to make this change.

**Comment by tellypresence** [comment]
@bensewell Yes, please prepare a PR and associate with this issue, thank you

**Comment by tellypresence** [comment]
@bensewell Closing for now, please reopen when work on PR begins
